src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -2390,6 +2392,18 @@ private void HandleReferences(RavenTransaction tx)
             _updateReferenceLoadWarning = false;
         }
 
+        private void HandleMismatchedReferences()
+        {
+            if (CurrentIndexingScope.Current._mismatchedReferencesHandler.MismatchedReferences.Count == 0) 
```

Comment:
Instead of checking `.MismatchedReferences.Count` you can add a property `IsEmpty` and you can make `MismatchedReferences` private

Reply:
Done